evc-sumo/Dockerfile

```diff
@@ -32,7 +33,7 @@ COPY --chown=carma:carma /docker ./docker
 COPY --chown=carma:carma /src ./src
 COPY --chown=carma:carma /test ./test
 
-RUN docker/install.sh ${EVC_TOKEN}
+RUN docker/install.sh evc_token
```

Comment:
I think you need to reference it as `$evc_token` still, this is just passing in the raw string "evc_token"